lib/healthcheck/api_tokens.rb

```diff
@@ -0,0 +1,47 @@
+module Healthcheck
+  class ApiTokens
+    WARNING_THRESHOLD = 1.year.to_i
+    CRITICAL_THRESHOLD = 1.month.to_i
```

Comment:
I wonder if these are a bit too high - I think something that is a year away wouldn't be enough to inspire me to do something soon.

I'd suggest maybe 1 month warning and 1 week critical?

Reply:
Going for 2 months, 1 month, as discussed.